### PR TITLE
Comment about display problems and fix with older kernel

### PR DESCRIPTION
**Pull request by mmolhoek**
Comment about display problems and fix with older kernel

PS: now from its own branch....sry

### CODE REVIEW

**Comment by mmolhoek** [comment]
I have read the CLA Document and I hereby sign the CLA

File: source/tutorials/installing-nixos-on-a-raspberry-pi.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ $ nix-shell -p wget zstd
 :::{note}
 You can pick a newer image by going to [Hydra job](https://hydra.nixos.org/job/nixos/trunk-combined/nixos.sd_image.aarch64-linux),
 clicking on a build and copying the link to the build product image.
+There have been reported [some problems](https://github.com/NixOS/nixpkgs/issues/179701) with the latest sd images on the pi 4, where it turns of the display while booting up.
```
Comment (fricklerhandwerk) [inline source/tutorials/installing-nixos-on-a-raspberry-pi.md:45]:
```suggestion

There have been reported [some problems](https://github.com/NixOS/nixpkgs/issues/179701) with the latest SD images on the Pi 4, where it turns off the display while booting up.
```

Open a new paragraph for each topic. Lumping everything together makes the text harder to scan and parse, and most readers don't actually read.

Please run basic spell check before submitting PRs. Use the official spelling of product names.

File: source/tutorials/installing-nixos-on-a-raspberry-pi.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ $ nix-shell -p wget zstd
...
+If you are experiencing such problems, please use [this image](https://hydra.nixos.org/build/134720986/download/1/nixos-sd-image-21.03pre262561.581232454fd-aarch64-linux.img.zst) instead until it is fixed in the newest sd images.
```
Comment (fricklerhandwerk) [inline source/tutorials/installing-nixos-on-a-raspberry-pi.md:46]:
```suggestion
If you are experiencing such problems, please use [the most recent working build](https://hydra.nixos.org/build/134720986/download/1/nixos-sd-image-21.03pre262561.581232454fd-aarch64-linux.img.zst) instead until the issue is fixed.
```

Name links in a way that is self-explanatory. The link should be readable in isolation. 

Also in general, don't just point people to "this random-sounding thing", that doesn't exactly inspire confidence and requires additional scrutiny. Make evident why this is the correct approach to follow. In this case it's only a matter of telling what it is you're pointing to.

**Review by fricklerhandwerk — CHANGES_REQUESTED** [review]
Thanks for the fix! Please fix the errors though. We try to establish a high standard of quality to improve Nix's reputation with regard to documentation.

Please also consider the [contribution guides](https://nix.dev/contributing#contribute-to-documentation), where all of that is detailed. I'll be happy to take feedback if these guides are approachable and helpful to make high quality changes with reasonable effort.

**Pull request merged**